packages/sanity-plugin-nacelle-input/README.md

```diff
@@ -91,33 +91,53 @@ Realistically, you probably want to restrict the component to _either_ products
 ```js
 // example: products ONLY
 {
-  name: 'productHandle',
+  name: 'productReference',
   title: 'Product',
-  type: 'nacelleData',
+  type: 'nacelleReference',
   options: {
     dataType: ['products']
   }
 }
 ```
 
-### Using This Data in Your Frontend Project
+### Data Shape
+
+The custom input component will store a reference object with the following format:
+
+```js
+{
+  type: 'NacelleReference',
+  referenceType: 'PRODUCT', // PRODUCT or COLLECTION
+  nacelledEntryId: '123455'
+  handle: 'blue-shirt'
+  locale: 'en-US'
+}
+```
 
-Since this custom input component just stores the `handle` of a particular product or collection, you'll use the [Nacelle Storefront SDK](https://nacelle.com/docs/querying-data/storefront-sdk?nacelle=v2&storefrontSdk=1.0.3) to fetch the associated product or collection object.
+## Backwards compatibility
 
-#### Product
+A backwards compatible custom input called `nacelleData` is also inlcuded. This input stores only the `handle` of a particular product or collection.
```

Comment:
I think we should be explicit about what this is backwards-compatible with:

```suggestion
For projects that were using [@nacelle/sanity-plugin-pim-linker](https://www.npmjs.com/package/@nacelle/sanity-plugin-pim-linker) for product/collection references, we provide a backward-compatible custom input called `nacelleData`. This input stores only the `handle` of a particular product or collection rather than a structured `nacelleReference` object.
```